src/main/java/com/amazon/opendistroforelasticsearch/sql/plugin/RestSqlAction.java

```diff
@@ -141,7 +142,8 @@ private void executeSqlRequest(final RestRequest request, final QueryAction quer
                                    final Client client, final RestChannel channel) throws Exception {
         if (isExplainRequest(request)) {
             final String jsonExplanation = queryAction.explain().explain();
-            channel.sendResponse(new BytesRestResponse(OK, "application/json; charset=UTF-8", jsonExplanation));
+            final String jsonPretty = new JsonPrettyFormatter().format(jsonExplanation);
```

Comment:
There seem to be quite a lot of code to have formatted json. 
WIll 

    new JSONObject(jsonExplanation).toString(4) work here? It could simplify code a lot

Reply:
That's what I've tried in the first place. The the outcome was not standard json format, with some missing returns. So I turned to the internal lib Xcontent.

Reply:
@chloe-zh could you provide some examples?

Reply:
Yes sure!
An example output of `new JSONObject(jsonExplanantion).toString(4)`:
```
{
    "size": 0,
    "stored_fields": "add_1",
    "script_fields": {"add_1": {
        "ignore_failure": false,
        "script": {
            "source": "def add_2 = doc['time_operating_in_hours'].value + 1;return add_2;",
            "lang": "painless"
        }
    }},
    "from": 0,
    "_source": {
        "excludes": [],
        "includes": ["script"]
    },
    "aggregations": {"equipment_id.keyword": {"terms": {
        "shard_min_doc_count": 0,
        "field": "equipment_id.keyword",
        "size": 200,
        "show_term_doc_count_error": false,
        "min_doc_count": 1,
        "order": [
            {"_count": "desc"},
            {"_key": "asc"}
        ]
    }}}
}
```

An example output of JsonPrettyFormatter (XContent):
```
{
  "from" : 0,
  "size" : 0,
  "_source" : {
    "includes" : [
      "script"
    ],
    "excludes" : [ ]
  },
  "stored_fields" : "add_1",
  "script_fields" : {
    "add_1" : {
      "script" : {
        "source" : "def add_2 = doc['time_operating_in_hours'].value + 1;return add_2;",
        "lang" : "painless"
      },
      "ignore_failure" : false
    }
  },
  "aggregations" : {
    "equipment_id.keyword" : {
      "terms" : {
        "field" : "equipment_id.keyword",
        "size" : 200,
        "min_doc_count" : 1,
        "shard_min_doc_count" : 0,
        "show_term_doc_count_error" : false,
        "order" : [
          {
            "_count" : "desc"
          },
          {
            "_key" : "asc"
          }
        ]
      }
    }
  }
}
```

Reply:
Thanks for providing an example, that definitely explains everything. Let's keep the code as is now